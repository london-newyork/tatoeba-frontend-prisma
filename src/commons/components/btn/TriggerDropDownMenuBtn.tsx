import React from 'react';
type TriggerDropDownMenuBtnProps = {
  onClick: () => void;
  children: React.ReactNode;
  className: string;
};
export const TriggerDropDownMenuBtn = ({ onClick, children, className }: TriggerDropDownMenuBtnProps) => {
  return (
    <button type="button" className={className} onClick={onClick}>
      {children}
    </button>
  );
};
