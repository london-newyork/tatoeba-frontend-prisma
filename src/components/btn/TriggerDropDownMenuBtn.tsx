import React from 'react';
type TriggerDropDownMenuBtnProps = {
  onClick: () => void;
  children: React.ReactNode;
  style: string;
};
export const TriggerDropDownMenuBtn = ({
  onClick,
  children,
  style,
}: TriggerDropDownMenuBtnProps) => {
  return (
    <button className={style} onClick={onClick} type='button'>
      {children}
    </button>
  );
};
