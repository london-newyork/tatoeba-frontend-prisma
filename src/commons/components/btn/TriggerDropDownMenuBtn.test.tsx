import { render } from '@testing-library/react';
import { TriggerDropDownMenuBtn } from './TriggerDropDownMenuBtn';

test('snapshot testing', () => {
  const { asFragment } = render(
    <TriggerDropDownMenuBtn
      className=""
      onClick={() => {
        undefined;
      }}
    >
      テキスト
    </TriggerDropDownMenuBtn>
  );
  expect(asFragment()).toMatchSnapshot();
});
