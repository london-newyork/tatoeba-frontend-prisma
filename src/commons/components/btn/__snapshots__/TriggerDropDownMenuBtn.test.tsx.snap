// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing 1`] = `
<DocumentFragment>
  <button
    class=""
    type="button"
  >
    テキスト
  </button>
</DocumentFragment>
`;
